lib/actions/apiV2.js

```diff
@@ -972,7 +972,7 @@ export function routingQuery(searchId = null, updateSearchInReducer) {
               }
 
               // Filter itineraries to collapse short names and hide unnecessary errors.
-              const withCollapsedShortNames = filteredItineraries.map(
+              const withCollapsedShortNames = filteredItineraries?.map(
```

Comment:
Can you also add a `itineraries?.filter` check at line 966 above?

Reply:
Good catch thanks!